### DISCUSSION TITLE
Joystick setups!

### DISCUSSION THREAD

**Discussion by jjburton** [jjburton/cgmTools#186] [Ideas]
It's time to start joystick setups. We need to figure out a good base set of control setups. 

### Branch Notes
features.joystick

### FACEUTILS
- 🆕 | New module
- SDKGroups_pushTo
- SDKGroups_select
- SDKGroups_verify
- poseBuffer | class handler for setting up our poseBuffers

### CGMMETA
- cgmObject.doCreateAt | added connectAs KW to wire a created object to the source

### Mel
- seShapeTaper | Scott Englert did a bug fix for maya changes. Thanks Scott!

**Comment by jjburton** [comment]
Bo did a setup with LBS last year here:

- Brow Left/Right
- Eye Left/Right
- Blink Left/Right
- Jaw
- corners (would need a split)

https://twitter.com/CgMonastery/status/1348394421851729921

**Comment by benngarnish** [comment]
![97124746-E782-4270-9E43-A1FF77436632](https://user-images.githubusercontent.com/1122587/104158391-2ea34400-53e5-11eb-8406-783f290a0c7b.png)

This is what I have so far. 
I propose we turn on and off sliders that a user doesn’t need when we build the rig.

**Reply by jjburton** [reply]
@benngarnish  | Do you have a version of this wired to a face to test? Wondering if we have a facs face and then a lighter keyframe face with different options.

I wanna try to use the cage shapes on drake's face as well to see if we can sucessfully just use that to drive some shapes on the regular build faces. I started back on facs last night.

**Reply by jjburton** [reply]
Looking at just joystick file:

- Is nose flare/compress not a facs shape? Or just not a pose of this package? Maybe that's what nose sneer vs full sneer is?
- Eye | is center control for both and the other for each? If so, I'd think just each eye would be more intuitive than an extra control. Am I missing something?
- I like the general layout here but would want to tweak a little I think.
- Would roll in/out not be better split left and right?
- Is our thinking we'd forego tongue, jaw eye regular controls and ONLY have afacs or a combo more like Morphy's kids? What is most useful to facs users? Maybe a single ui is the way that's best used?
-

**Comment by jjburton** [comment]
Old Morpheus 2 Mock up

https://twitter.com/CgMonastery/status/1350494080405536770

![image](https://user-images.githubusercontent.com/48218358/104818363-bfff2580-57ec-11eb-903a-5e1567bf6bdc.png)

Pass on this
https://mobile.twitter.com/CgMonastery/status/1351019319283642370

**Comment by jjburton** [comment]
https://www.artstation.com/artwork/2xgvYK

Great example
